pkg/infrabin/http_test.go

```diff
@@ -284,8 +287,8 @@ func TestProxyHandler(t *testing.T) {
 	}
 
 	rr := httptest.NewRecorder()
-	is := &InfrabinService{Config: &Config{EnableProxyEndpoint: true}}
-	handler := NewHTTPServer("test", "", RegisterInfrabin("/", is)).Server.Handler
+	is := &InfrabinService{}
+	handler := NewHTTPServer("test", RegisterInfrabin("/", is)).Server.Handler
```

Comment:
Now this is a normal server we can use the helper function.
```suggestion
	handler := newHTTPInfrabinHandler()
```